vnext/Microsoft.ReactNative.Cxx/NativeModules.h

```diff
@@ -75,6 +75,16 @@
 // It can be an instance or static method.
 #define REACT_CONSTANT_PROVIDER(method) INTERNAL_REACT_MEMBER_2_ARGS(ConstantMethod, method)
 
+// REACT_GET_CONSTANTS(method)
+// Arguments:
+// - method (required) - the method name the macro is attached to.
+//
+// REACT_GET_CONSTANTS annotates a method that defines constants.
+// It must have no parameter.
+// It must return a REACT_STRUCT decorated struct, fields become constants.
```

Comment:
>It must return a REACT_STRUCT decorated struct [](http://example.com/codeflow?start=3&length=46)

Can we easy this requirement to be a type that has ReadValue method defined for? The REACT_STRUCT defines this method for a type.

Reply:
I don't think so, because the return value should be a set of constant values. Maybe map could also do that, but I don't think other types could do.

Reply:
What I am using is the `WriteProperties` function

Reply:
It is OK to say that it must be a struct and we should probably add a static_assert for it. I am just saying that we can define WriteValue for a struct using three different ways: explicit WriteValue method, REACT_STRUCT, and providing GetStructInfo method ([https://github.com/microsoft/react-native-windows/blob/c7292997f95862d5c3bd52ce1ff71936211bcd4f/vnext/Microsoft.ReactNative.Cxx.UnitTests/JSValueReaderTest.cpp#L35](https://github.com/microsoft/react-native-windows/blob/c7292997f95862d5c3bd52ce1ff71936211bcd4f/vnext/Microsoft.ReactNative.Cxx.UnitTests/JSValueReaderTest.cpp#L35)). 
All three methods must be valid.

Reply:
No code change is required - it is just the comment wording change. The WriteProperties already behaves the way I had described, except for checking that the type is a class/struct.